frontend/src/main.jsx

```diff
@@ -1,7 +1,20 @@
+import { ApolloClient, ApolloProvider, InMemoryCache } from "@apollo/client";
 import React from "react";
 import ReactDOM from "react-dom/client";
 import "./default.css";
 
+const client = new ApolloClient({
+  uri:
+    //in Vite, use special object `import.meta.env` to access enviroment variables
+    import.meta.env.MODE === "production"
+      ? //created environment variables must be prefixed by VITE
+        import.meta.env.VITE_API_URI
```

Comment:
Can you add an example environment variable file, similar to how we have one in the backend directory?